code/modules/lighting/static_lighting_area.dm

```diff
@@ -15,3 +15,19 @@ GLOBAL_DATUM_INIT(fullbright_overlay, /mutable_appearance, create_fullbright_ove
 //Non static lighting areas.
 //Any lighting area that wont support static lights.
 //These areas will NOT have corners generated.
+
+///regenerates lighting objects for turfs in this area
```

Comment:
Can you mention that this proc is mainly for VV so it's not misused please.

Reply:
Adressed